lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
You are right, when I add your change from the DataGenerator, it fails without `disconnect_contacts` ... We probably need better tests, because without the additional double-sided contact, it fails without `.reload`. I created #7538 to fix that, that should now cover both problems. 

I need to sleep now, because I'm tired and I need to work tomorrow, but I'll have a closer look at this tomorrow, to improve the tests. Or if you have the time to improve the tests in your PR, you can also do it.

@denschub: can you merge #7538 as soon as travis is green?